src/util/get-config-files.ts

```diff
@@ -45,26 +47,25 @@ export async function getConfigFiles(
       path,
     });
 
-    return [requestedRepoFile, defaultRepoConfig];
+    files.push(defaultRepoConfig);
```

Comment:
Instead of returning I just push the file from the `.github` repo in the `files` array so it will become part of the resolve chain.